tests/test_prior.py

```diff
@@ -75,3 +71,24 @@ def test_map_contains_log_det_jacobian():
     assert np.allclose(- m1.neg_log_marginal_likelihood().numpy(),
                        - m2.neg_log_marginal_likelihood().numpy() + m1.log_scale), \
             "MAP objective should differ by log|Jacobian| of the transform"
+
+
+def get_GPMC_model_params():
+    kernel = gpflow.kernels.Matern32()
+    likelihood = gpflow.likelihoods.Gaussian()
+    data = [np.arange(5), np.arange(5)]
+    return data, kernel, likelihood
+
+
+@pytest.mark.parametrize('model_class, args', [
+    (gpflow.models.GPMC, get_GPMC_model_params()),
+    #(gpflow.models.SGPMC, get_SGPMC_model_params()) # Fails due to inducing_variable=None bug
+    ])
+def test_V_prior_dtypes(model_class, args):
```

Comment:
```suggestion
def test_v_prior_dtypes(model_class, args):
```

Reply:
the variable is called `V`, so I think it's actually clearer to call it `test_V_prior_dtypes`.